RealmTasks Android/app/src/main/java/io/realm/realmtasks/list/ItemViewHolder.java

```diff
@@ -191,12 +199,16 @@ public void setIconBarAlpha(float alpha) {
         iconBar.setAlpha(alpha);
     }
 
-    public void preCompleted() {
+    public void changeBackgroundColorIfNeeded() {
+        if (backgroundColorVersion == completedVersion) {
+            return;
+        }
         if (getCompleted()) {
```

Comment:
you can access the boolean directly for speed like you did in https://github.com/realm/RealmTasks/blob/6e4f8ac6d73e49712855769bcd69ec37bce5a6e7/RealmTasks%20Android/app/src/main/java/io/realm/realmtasks/list/ItemViewHolder.java#L77